docs/fundamentals/executor/executor-api.md

```diff
@@ -294,6 +294,29 @@ Text: hello world
 ```
 
 
+You can even pull an `Executor` from the hub and use it directly as a python object.
```

Comment:
Maybe we need a list of the methods from Executor API, as in the Flow. and link about Hub to give context

Reply:
What do you mean by a list of method ? Which part of the Flow documentation are you referring to ?